### PR TITLE
Added tax and taxtype entities to add and delete test data.

### PR DESCRIPTION
**Pull request by meghab2**
Fixed gsm and Pricelist values #3 #2 fixed.

### CODE REVIEW

File: scripts/create_test_data.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 import random
 from random import randint
 
+
```
Comment (vritant) [inline scripts/create_test_data.py]: no need for extra white space lines, please delete

File: scripts/create_test_data.py [hunk 2]
```diff
@@ -45,13 +53,30 @@ def get_code(name):
     for name in names:
         code+=name[:1]
     return code
+
```
Comment (vritant) [inline scripts/create_test_data.py]: same as above, extra whitespaces

File: scripts/create_test_data.py [hunk 3]
```diff
@@ -45,13 +53,30 @@ def get_code(name):
...
         code+=name[:1]
     return code
+    
+
 
 def create_price_list():
     pl = swagger_client.PriceList()
-    pl.wef_date_from = '-'.join(['2016',random.choice(month),random.choice(day)])
-    pl.wef_date_to = '-'.join(['2016',random.choice(month),random.choice(day)])
+    pl.wef_date_from = '-'.join(['2016',random.choice(month_start),random.choice(day)])
```
Comment (vritant) [inline scripts/create_test_data.py:54]: :thumbsup:

File: scripts/create_test_data.py [hunk 4]
```diff
@@ -99,13 +125,14 @@ def create_mill():
     for i in range(0, size):
         sgs = swagger_client.SimpleGsmShade()
         sgs.shade = random.choice(shade)
-        sgs.min_gsm = randint(0, 500)
-        sgs.max_gsm = randint(0, 500)
+        sgs.min_gsm = randint(30, 50)
```
Comment (vritant) [inline scripts/create_test_data.py:119]: :+1:

**Comment by vritant** [comment]
@meghab2 I have maintained a single empty line space between code blocks in both the scripts.
Please maintain that.

File: scripts/create_test_data.py [hunk 5]
```diff
@@ -99,13 +125,14 @@ def create_mill():
...
+        sgs.max_gsm = randint(51, 500)
         sgs.mill = mill
         print "Creating SimpleGsmShade: %s(%s-%s) for Mill: %s" % (sgs.shade, sgs.min_gsm, sgs.max_gsm, mill.name)
-        all_sgs.append(sgs_api.create_simple_gsm_shade_using_post(sgs))
-    return mill_api.get_mill_using_get(mill.id)
-
+        sgs_api.create_simple_gsm_shade_using_post(sgs)
```
Comment (vritant) [inline scripts/create_test_data.py]: This breaks price creation. look up use of all_sgs later in the script.

File: scripts/create_test_data.py [hunk 6]
```diff
@@ -99,13 +125,14 @@ def create_mill():
...
+    return mill
```
Comment (vritant) [inline scripts/create_test_data.py]: This breaks price creation too. if you dont refresh the mill, it wont have qualities to create prices for.

**Comment by vritant** [comment]
@meghab2 I am done with review. please fix these issues, and I will merge.

File: scripts/delete_test_data.py [hunk 7]
```diff
@@ -44,6 +46,14 @@
     for group in groups:
         print "Deleting Customer Group: %s" % group.name
         customer_group_api.delete_customer_group_using_delete(group.id)
+        
+while True:
+    tax = tax_api.get_all_taxs_using_get()
```
Comment (vritant) [inline scripts/delete_test_data.py]: use plural variable please

File: scripts/delete_test_data.py [hunk 8]
```diff
@@ -68,3 +78,14 @@
     for customer in customers:
         print "Deleting Customer: %s" % customer.name
         customer_api.delete_customer_using_delete(customer.id)
+        
+        
+while True:
+    taxtype = taxtype_api.get_all_tax_types_using_get()
```
Comment (vritant) [inline scripts/delete_test_data.py]: use plural variable please

File: scripts/create_test_data.py [hunk 9]
```diff
@@ -45,13 +53,30 @@ def get_code(name):
...
 def create_price_list():
     pl = swagger_client.PriceList()
-    pl.wef_date_from = '-'.join(['2016',random.choice(month),random.choice(day)])
-    pl.wef_date_to = '-'.join(['2016',random.choice(month),random.choice(day)])
+    pl.wef_date_from = '-'.join(['2016',random.choice(month_start),random.choice(day)])
+    pl.wef_date_to = '-'.join(['2016',random.choice(month_end),random.choice(day)])
     pl = pricelist_api.create_price_list_using_post(pl)
     return pl
+    
+def create_tax_type(plist):
```
Comment (vritant) [inline scripts/create_test_data.py]: please rename to create taxes.

File: oms [hunk 10]
```diff
@@ -0,0 +1 @@
+Subproject commit af0a7f84b405bafde613660a5b5175ca7c65c09c
```
Comment (vritant) [inline oms]: why submodule? was this intentional?

**Comment by vritant** [comment]
doesnt show up in this chain, but why submodule? was that intentional?

File: scripts/create_test_data.py (hunk 2 above)
Comment (meghab2) [inline scripts/create_test_data.py]: Done

File: scripts/create_test_data.py (hunk 6 above)
Comment (meghab2) [inline scripts/create_test_data.py]: Sorry. I was trying to do create for derived gsm in the same function. Missed to place the code back

File: scripts/create_test_data.py (hunk 9 above)
Comment (meghab2) [inline scripts/create_test_data.py]: done

File: oms (hunk 10 above)
Comment (meghab2) [inline oms]: This was a typo and now i have rectified it by deleting that folder.

File: scripts/delete_test_data.py (hunk 8 above)
Comment (meghab2) [inline scripts/delete_test_data.py]: done

**Comment by vritant** [comment]
@meghab2 added some clean up after merge, please look at d9280f6

**Comment by meghab2** [comment]
Make sense.

**Pull request merged**